extensions/tags/js/src/forum/extend.ts

```diff
@@ -1,14 +1,22 @@
 import app from 'flarum/forum/app';
 import Extend from 'flarum/common/extenders';
 import IndexPage from 'flarum/forum/components/IndexPage';
+import Discussion from 'flarum/common/models/Discussion';
 import DiscussionTaggedPost from './components/DiscussionTaggedPost';
 import TagsPage from './components/TagsPage';
+import Tag from '../common/models/Tag';
+
+import commonExtend from '../common/extend';
 
 export default [
+  ...commonExtend,
+
   new Extend.Routes()
     .add('tags', '/tags', TagsPage)
     .add('tag', '/t/:tags', IndexPage)
     .helper('tag', (tag) => app.route('tag', { tags: tag.slug() })),
 
   new Extend.PostTypes().add('discussionTagged', DiscussionTaggedPost),
```

Comment:
I left a comment above, but this is what I was thinking about; we have `.add('name', Class)` for PostTypes, and `(Class).register('name')` for models. This feels inconsistent